.doc_gen/metadata/iam_metadata.yaml

```diff
@@ -718,6 +776,18 @@ iam_CreatePolicy:
   synopsis: create an &IAM; policy.
   category:
   languages:
+    .NET:
+      versions:
+        - sdk_version: 3
+          github: dotnetv3/IAM
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - IAM.dotnetv3.CreatePolicyExample
+            - description:
+              snippet_tags:
+                - IAM.dotnetv3.CreatePolicyAsync
```

Comment:
Include only one of these snippets--I'd pick the second because it's the tested one, but it's your call.

Reply:
Kept the second example.